src/pudl/output/epacems.py

```diff
@@ -103,3 +109,48 @@ def get_plant_years(plant_ids, pudl_out):
         .query("plant_id_eia in @plant_ids")
         .report_date.dt.year.unique()
     )
+
+
+def epacems(
+    states: Optional[Sequence[str]] = None,
+    years: Optional[Sequence[int]] = None,
+    columns: Optional[Sequence[str]] = None,
+    epacems_path: Optional[Path] = None,
+) -> dd.DataFrame:
+    """Load EPA CEMS data from PUDL with optional subsetting.
+
+    Args:
+        states (Optional[Sequence[str]], optional): subset by state abbreviation. Defaults to None (gets all states).
+        years (Optional[Sequence[int]], optional): subset by year. Defaults to None (gets all years).
+        columns (Optional[Sequence[str]], optional): subset by column. Defaults to None (gets all columns).
+        epacems_path (Optional[Path], optional): path to parquet dir. By default it automatically loads the path from pudl.workspace
+
+    Returns:
+        dd.DataFrame: epacems data
+    """
+    if states is None:
+        states = list(pudl.constants.us_states.keys())  # all states
```

Comment:
I think this may actually fail, since the `us_states` constant includes territories, AK, HI, etc. You should use the years and states that are enumerated in the `pudl.constants.working_partitions` dictionary to determine whether the states/years that have been requested are valid, and to obtain a complete list if you want to process everything when there's no input given here.

Reply:
dd.read_parquet returns an empty dataframe if you try to select a non-existent partition. Trying to select both existing and non-existing partitions returns only the existing ones. For example, `epacems(states=['asdf'])` returns an empty dataframe, and `epacems(states=['ID', 'asdf'])` returns only Idaho.
Is that fine, or should we prefer an exception?

Reply:
As a side note, there is CEMS data for a plant in Alaska, but we have hardcoded it out of our dataset by dropping 'AK' somewhere in the pipeline. I held off on doing anything about that until integrating the crosswalk, at which point there will be a handful of other tweaks to CEMS as well.

Reply:
Looks like AK just showed up in 2019 and 2020, and we're archiving the data, so if I just remove it from the list of excluded territories in the `EPACEMS_STATES` constant it should get integrated.

Reply:
And then there's one lonely year of data for Puerto Rico in 2015 😢 🇵🇷